se/Mortgage/v0/schema/definitions/Applicant.yaml

```diff
@@ -0,0 +1,185 @@
+"$id": "https://open-broker.org/se/Mortgage/v0/schema/definitions/Applicant"
+"$schema": "http://json-schema.org/draft-07/schema#"
+
+title: Applicant
+description: |
+  Describe the attributes of an applicant, such as housing situation,
+  employment details, financial information, marital status and citizenships.
+
+type: object
+additionalProperties: false
+required:
```

Comment:
Would "education" be of interest to include? Showing the highest finished education of the applicant. 
Example: primary school (grundskola), secondary school (gymnasium), university short (1-3 years), university long (4+ years).

Reply:
It is possible, I should research this.

Reply:
It seems like this is not requested by any bank.